CONTRIBUTING.md

```diff
@@ -180,6 +183,32 @@ To ensure a clean build, you may also run this npm script before running the `in
 npm run cleanAll
 ```
 
+### Proxy testing
+
+For testing the extension with a proxy server, it is recommended that you use the reverse
+proxy server included in the Java Client project for proxy testing. To start that server,
+simply run the following command in the root directory of the Java Client project. That
+command will start the reverse proxy server in blocking mode listening on port 8020. It
+will then forward requests to port 8020 based on the custom mappings.
+
+```
+./gradlew runBlockingReverseProxyServer -PrpsCustomMappings=/mlxprs/manage,8059,/mlxprs/rest,8055,/mlxprs/test,8054
```

Comment:
This is good, but feel free to add these to the Java Client too, there's no harm in it having mappings that the java-client-api tests don't need.